Userland/Utilities/CMakeLists.txt

```diff
@@ -73,9 +73,9 @@ target_link_libraries(comm LibMain)
 target_link_libraries(config LibConfig LibMain)
 target_link_libraries(copy LibGUI LibMain)
 target_link_libraries(cp LibMain)
-target_link_libraries(cpp-lexer LibMain)
-target_link_libraries(cpp-parser LibMain)
-target_link_libraries(cpp-preprocessor LibMain)
+target_link_libraries(cpp-lexer LibCpp LibMain)
+target_link_libraries(cpp-parser LibCpp LibGUI LibMain)
+target_link_libraries(cpp-preprocessor LibCpp LibGUI LibMain)
```

Comment:
Judging from the includes, these don't need to link against LibGUI... Likely just a copy & paste mistake from when the new entries were added.

Reply:
i followed the blame, that used to be a SyntaxHighlighter https://github.com/SerenityOS/serenity/commit/ff2438e0ceac3b99248748f6a5caf49c223e041d#diff-bc6ec6527c8e76822c38c34bc969f52595dbdbda8909eca0abf1763695a7c322L51
`public GUI::SyntaxHighlighter`